spec/forms/applicants/basic_details_form_spec.rb

```diff
@@ -66,8 +66,21 @@
       end
     end
 
-    context "with dob in the future" do
-      let(:attributes) { attributes_for(:applicant).merge(date_of_birth: 3.days.from_now) }
+    context "with date_of_birth in the future" do
+      let(:attributes) { attributes_for(:applicant).merge(date_of_birth: Date.tomorrow) }
+
+      it "does not persist model" do
+        expect { subject.save }.not_to change(Applicant, :count)
+      end
+
+      it "errors to be present" do
```

Comment:
This reads a bit weirdly... how about 
> it "generates errors"

So that the rspec output will read
> with date_of_birth in the future it generates errors

?

Reply:
sure, was a copy and paste, from previous which i will amend too - i was not keen on the language of the spec either